media-proxy/include/mtl.h

```diff
@@ -40,20 +40,6 @@ extern "C" {
 #define SCH_CNT 3
 #define TASKLETS 1000
 
-enum direction {
-    TX,
-    RX
-};
-
-typedef struct {
-    uint8_t is_master;
-    char app_name[32];
-    char interface_name[32];
-    uint32_t interface_id;
-    char socket_path[108];
-    uint32_t m_session_count;
-} memif_ops_t;
```

Comment:
I moved memif_ops_t from mtl.h to shm_memif.h, because I needed this structure for RDMA as well, not only for mtl.c. It is more connected to memif then mtl. 
I moved `enum direction` to libfabric.ep, but it was not the best places. I will move it to utils.

Reply:
enum direction is now in utils.h. 
Not totally sure if this file it the best, but I needed something common to not cause circular dependency as enum direction is needed in many files